R/as-data-frame.R

```diff
@@ -112,6 +112,11 @@ csvToDataFrame <- function (csv_df, crdf) {
             cp <- columnParser("categorical")
             sub_a <- aliases(subvariables(v))
             return(structure(lapply(csv_df[sub_a], cp, v, mode), .Names=sub_a))
+        } else if (is.Numeric(v)) {
+            # We use this to coerce the missing entries to NA, so that warning is
```

Comment:
This comment should be more explicit about what the data looks like such that coercion is needed.